api/v1/controllers/EventController.js

```diff
@@ -0,0 +1,150 @@
+var bodyParser = require('body-parser');
+var _Promise = require('bluebird');
+
+var errors = require('../errors');
+var services = require('../services');
+var config = require('../../config');
+
+var middleware = require('../middleware');
+var requests = require('../requests');
+var utils = require('../utils');
+var roles = require('../utils/roles');
+
+var EventService = require('../services/EventService');
+
+var logger = require('../../logging');
+
+var router = require('express').Router();
+
+function getEvents(req, res, next) {
+        var updated = req.query.lastUpdated || 0;
+        return services.EventService.findByUpdated(updated)
+                .then(function(result) {
+                        res.body = result.length == 0 ? [] : result.map(function (obj) {
+                                return obj.toJSON();
+                        });
+                        next();
+                        return null;
+                });
```

Comment:
This is intended, since it should just return an empty array if there are no events. So I don't throw an error when there are no events, but instead I return an empty array if it makes sense.